Libraries/LibWeb/HTML/Navigator.h

```diff
@@ -28,7 +29,8 @@ class Navigator : public Bindings::PlatformObject
     , public NavigatorIDMixin
     , public NavigatorLanguageMixin
     , public NavigatorOnLineMixin
-    , public StorageAPI::NavigatorStorage {
+    , public virtual StorageAPI::NavigatorStorage
+    , public virtual Credentials::NavigatorCredentialsContainerMixin {
```

Comment:
changing the this_foo_object method name to reflect the interface would avoid the need for virtual inheritance here (which I don't think we are using... anywhere, actually :o )